components/content_settings/core/browser/brave_content_settings_pref_provider.cc

```diff
@@ -119,9 +119,18 @@ bool IsActive(const Rule& cookie_rule,
 
 // static
 void BravePrefProvider::CopyPluginSettingsForMigration(PrefService* prefs) {
+  if (!prefs->HasPrefPath("profile.content_settings.exceptions.plugins")) {
```

Comment:
Nit: I think you can drop the braces here, since it's a single line condition followed of a single line statement

Reply:
Personally I find it more readable with braces, so I would leave it as is